### ISSUE TITLE
Beta 2022.12 Broke Our Groceries

### ISSUE THREAD

**Issue by bschatzow** [ljmerza/ha-our-groceries#29]
## Version of the custom_component
1.4.1

## Configuration

```yaml

Add your logs here.
Error Log:
Package ourgrocery setup failed. Component ourgroceries Requirements for ourgroceries not found: ['ourgroceries==1.4.1'].

```

## Describe the bug
Just updated to the December Beta and it broker our-groceries.  No changes other than this update.

## Debug log

```text

Add your logs here.

```

**Comment by chicknlil** [comment]
Just came to report the same thing.

My log is a bit more verbose:

```
Logger: homeassistant.util.package
Source: util/package.py:98
First occurred: 10:20:49 PM (3 occurrences)
Last logged: 10:21:01 PM

Unable to install package ourgroceries==1.4.1: ERROR: Cannot install ourgroceries==1.4.1 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts [notice] A new release of pip available: 22.3 -> 22.3.1 [notice] To update, run: pip install --upgrade pip
```

I did go into terminal and did the command it indicated, but that didn't solve the issue.

**Comment by dragonerix** [comment]
not woking with HA 2022.12 and pip 22.3.1

**Comment by jampez77** [comment]
I'm gettting the same issue with release 2022.12.0. Additionally it won't let me restart with this integration installed. I get this error:

`[281471536404016] The system cannot restart because the configuration is not valid: Integration error: ourgroceries - Integration 'ourgroceries' not found.
Traceback (most recent call last):`

  `File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 200, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1745, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1782, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/usr/src/homeassistant/homeassistant/components/hassio/__init__.py", line 529, in async_handle_core_service
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: The system cannot restart because the configuration is not valid: Integration error: ourgroceries - Integration 'ourgroceries' not found`

**Comment by kovrom** [comment]
Same issue here. Our Groceries 1.4.1 doesn't work with HA 2022.12.
`Unable to install package ourgroceries==1.4.1: ERROR: Cannot install ourgroceries==1.4.1 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts [notice] A new release of pip available: 22.3 -> 22.3.1 [notice] To update, run: pip install --upgrade pip`
Not trying to be rude or anything but just in case will tag Leonardo :) @ljmerza 
Thank you

**Comment by ljmerza** [comment]
should be fixed in v1.5.0
